workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/GameObjectRepresentationSystemHelper.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Unity.Entities;
+
+namespace Improbable.Gdk.Core.GameObjectRepresentation
```

Comment:
It's supposed to be in this weird limbo where we cannot yet move it into a feature module because we currently don't have codegen for feature modules, but we're basically pretending it's a feature module as much as we can.

Reply:
what kind of codegen do we need for it?

Reply:
Several files worth of classes for each component - Readers/Writers, Command Request and Response senders and receivers that can be [Require]d, GameObjectComponentDispatchers which take the reactive components that the Core put on entities showing the dispatcher ops and passing those along to lifecycle management (enable monobehaviour with writer on authority etc) and user callbacks registered on eg readers etc...

Reply:
makes sense! do we have a ticket for moving this to a separate feature module?

Reply:
Now we do, just made UTY-1131.

Reply:
Not being in its own module aside, why does it have to be in the `Core` namespace?

Reply:
Good question... I guess as a result of it being in core for now it seemed to make sense to have it named as a core sub namespace and noone really thought about it. This will make it slightly more annoying to move it out indeed, obviously should be different at that point.